packages/odyssey-react/src/components/Banner/Banner.test.tsx

```diff
@@ -0,0 +1,127 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import React from "react";
+import { render, fireEvent } from "@testing-library/react";
+import Banner from ".";
+
+const role = "status";
+const bannerTitle = "Banner title";
+const bannerContent = "Banner content";
+const bannerActions = "Banner.Children, <a href='https://www.okta.com'>with a link</a>";
+const bannerDismissButtonLabel = "Dismiss message";
+
+describe("Banner", () => {
+  it('renders the Banner', () => {
+    const { getByRole } = render(
+      <Banner visible={true} title={bannerTitle} content={bannerContent}>
+        {bannerActions}
+      </Banner>
+    );
+
+    expect(getByRole(role)).toBeInTheDocument();
+  });
+
+  it('renders the Banner title', () => {
+    const { getByText } = render(
+      <Banner visible={true} title={bannerTitle} content={bannerContent}>
```

Comment:
minor but JSX sugar allows you to write `visible={true}` as just `visible`

Reply:
Updated